mylang/private/toolchains_repo.bzl

```diff
@@ -0,0 +1,113 @@
+"""Create a repository to hold the toolchains
```

Comment:
Yeah I agree, in using this template a lot of the work is correctly removing all the toolchain bits you don't need.

However I think if we just document what to do, it will be hard to follow how to create a toolchain, and we'll end up with a lot of "code in comments" which is harder to keep it working.
If we make a separate template, they will diverge over time and we'll double the maintenance effort.
There are fancy systems for code templating like "cookiecutter" from python-land, where you can interactively ask the user what features to enable, but I don't think the SIG has time to invest in something fancier than just GitHub templates built-in feature.

So I'm torn about what is the better way to do this.